pwa/src/components/endpoints/endpointForm.tsx

```diff
@@ -111,22 +111,24 @@ export const EndpointForm: React.FC<EndpointFormProps> = ({ endpointId }) => {
     <form onSubmit={handleSubmit(onSubmit)}>
       <Card
         title={title}
-        cardHeader={function () {
+        cardHeader={() => {
           return (
             <div>
               <button
```

Comment:
This goes for every help button in forms (e.g. this one).

Because this is a `button`, the form will be sent when we click it. However, the form shouldn't be sending when a user clicks the help button.

To fix this, we can change the `button` tag to an `a` tag. 

You do need to test the lay-out afterwards!

Reply:
Note: this was previously taken care of by adding a `(e) => e.preventDefault()`. However, just changing it to an `a` tag would be a better solution.